custom-routes-vis/src/main/webapp/index.html

```diff
@@ -20,20 +20,30 @@ <h1>Custom Routes Visualization</h1>
       </div>
     </div>
     <div class="row">
-      <div class="col-sm-4 offset-sm-1">
+      <div class="col-sm-2 offset-sm-1">
         <h5>Origin Coordinates:</h5>
         <div id='origin-coordinates'></div><br>
         <button type="button" class="btn btn-success" onclick="showMarker('origin')" id="show-origin-marker">Place Marker</button>
         <button type="button" class="btn btn-danger hidden" onclick="hideMarker('origin')" id="hide-origin-marker">Delete Marker</button><br>
       </div>
-      <div class="col-sm-4">
+      <div class="col-sm-2">
         <h5>Destination Coordinates:</h5>
         <div id='destination-coordinates'></div><br>
         <button type="button" class="btn btn-success" onclick="showMarker('destination')" id="show-destination-marker">Place Marker</button>
         <button type="button" class="btn btn-danger hidden" onclick="hideMarker('destination')" id="hide-destination-marker">Delete Marker</button><br>
       </div>
-      <div class="col-sm-2">
-        <button type="button" class="btn btn-primary hidden" onclick="generateRoutes()" id="generate-routes">Generate Routes</button>
+      <div class="col-sm-3">
+        <select id="service-endpoint" class="form-control">
+          <option value="directions" selected>Directions API</option>
+          <option value="compute-routes">ComputeRoutes API(V1)</option>
+          <option value="compute-routes-alpha">ComputeCustomRoutes API(V1 alpha)</option>
+        </select>
+        <textarea id="api-key" class="form-control" placeholder="API key (defaults to use Directions API)" rows="1"></textarea>
```

Comment:
```suggestion
        <textarea id="api-key" class="form-control" placeholder="API Key (Defaults to use Directions API)" rows="1"></textarea>
```

Reply:
This placeholder as a whole is a little strange to me. The textarea is where they enter an API Key, so the "defaults" part should  be "Defaults to use Directions API Key", if that is what you mean. If you are talking about defaulting to the Directions API over the other APIs, I don't think it is necessary to have this considering that is what the dropdown signifies.

Reply:
Also is there a reason you are using textarea instead of input? The API key is not that long. I would think input is cleaner.

Reply:
placeholder fixed; using input instead